worker-tasks/sbt

```diff
@@ -0,0 +1,580 @@
+#!/usr/bin/env bash
+#
+# A more capable sbt runner, coincidentally also called sbt.
+# Author: Paul Phillips <paulp@improving.org>
+
```

Comment:
It's a convenience script for people who don't have sbt installed, because it downloads whatever it needs and configures SBT according to what version is stated in the `build.properties`. If the idea is that we always run this through a VM, and that VM has `sbt`, I don't see why we couldn't break from the pattern and just run the machine's sbt command.